docs/retrieve.txt

```diff
@@ -0,0 +1,456 @@
+            technology is given the inverse task of planting an idea into the mind of a CEO.
+
+            Title: Senna
+            Year: 2010
+            Runtime: 106
+            IMDB Rating: 8.6
+            IMDB Votes: 41904
+            Plot: A documentary on Brazilian Formula One racing driver Ayrton Senna, who won the
+            F1 world championship three times before his death at age 34.
+
+.. _laravel-retrieve-query-builder:
+
+Use Laravel Queries to Retrieve Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use Laravel's database query builder to run find operations instead of using Eloquent
+models. To run the database query, import the ``DB`` facade into your controller file and use
+Laravel's query builder syntax.
+
+This example uses Laravel's query builder to retrieve documents in which the value
+of the ``imdb.votes`` nested field is greater than ``1300000``.
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. code-block:: go
+
+         $movies = DB::connection('mongodb')
+               ->collection('movies')
+               ->where('imdb.votes', '>', 1300000)
+               ->get();
+
+   .. tab:: Run the Query
+      :tabid: query-run
+
+      To run the find operation and view the results, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
+               public function show()
+               {
+                  $movies = DB::connection('mongodb')
+                        ->collection('movies')
+                        ->where('imdb.votes', '>', 1300000)
+                        ->get();
+
+                  return view('browse_movies', [
+                     'movies' => $movies
+                  ]);
+               }
+            }
+
+         .. output::
+            :language: none
+            :visible: false
+
+            Title: The Shawshank Redemption
+            Year: 1994
+            Runtime: 142
+            IMDB Rating: 9.3
+            IMDB Votes: 1513145
+            Plot: Two imprisoned men bond over a number of years, finding solace and eventual redemption
+            through acts of common decency.
+
+            Title: The Dark Knight
+            Year: 2008
+            Runtime: 152
+            IMDB Rating: 9
+            IMDB Votes: 1495351
+            Plot: When the menace known as the Joker wreaks havoc and chaos on the people of Gotham, the
+            caped crusader must come to terms with one of the greatest psychological tests of his ability to
+            fight injustice.
+
+      Then, make the following changes to your Laravel Quick Start application:
+
+      - Add the ``use Illuminate\Support\Facades\DB`` use statement to your ``MovieController.php``
```

Comment:
Suggestion:

I think it could be good to match this with the other mention of this import in the section description since both versions of the example use it. The other instance currently reads:
"import the DB facade into your controller file"